rfp-proposals/ars-js-filecoin-explorer.md

```diff
@@ -0,0 +1,350 @@
+
+- 1 Global 
+  - General search function,you can search by block height or a CID etc., perherps the CID is a miner address, an account, a message, an actor,or an ask etc.
+
+- 2 Overview
+  - Best tipset,avg…，etc
+  - blockchain growing in real time
+  - Trend chart of storage miners
+    - abscissa: Time(24hour,Day,Week,Month,Year)
+    - ordinate:
+      - The line chart: 
+        - total power
+        - total collateral
+      - histogram:  total counts of active storage miner
+  - Trend chart of storage deals
+    - abscissa: Time(24hour,Day,Week,Month,Year)
+    - ordinate:
+      - The line chart:  dealed price (min,max,avg,start,end)
+      - histogram:  
+        - duration active clients counts
+        - duration dealed capacity
+  - Trend chart of retrieval miners
+    - abscissa: Time(24hour,Day,Week,Month,Year)
+    - ordinate:
+      - The line chart: total retrieval capacity
+      - histogram:  total counts of retrieva miner
+  - Trend chart of retrieval deals
+    - abscissa: Time(24hour,Day,Week,Month,Year)
+    - ordinate:
+      - The line chart:  dealed price (min,max,avg,start,end)
+      - histogram:  
+        - duration active clients counts
+        - duration dealed capacity
+  - Trend chart of  transfer
+    - abscissa: Time(24hour,Day,Week,Month,Year)
+    - ordinate:
+      - The line chart:  
+        - duration active  transfer amount
+      - histogram:  
+        - duration active account counts
+
+- 3 Block Chain
+  - Chain
+  - Actor
+
+
+
+- 4 StorageMarket
+  - Trend chart of storage miners ，power&collateral
+    - abscissa: Time(24hour,Day,Week,Month,Year)
+    - ordinate:
+      - The line chart: 
+        - total power
+        - total collateral
+      - histogram:  total counts of active storage miner
+  - Trend chart of storage miners counts
+    - abscissa: Time(24hour,Day,Week,Month,Year)
+    - ordinate:
+      - The line chart:
+        -  counts of total miners 
+        - counts of active miners 
+      - histogram:  
+        - duration counts of new miners 
+  - Trend chart of storage clients counts
+    - abscissa: Time(24hour,Day,Week,Month,Year)
+    - ordinate:
+      - The line chart: counts of total clients 
+      - histogram:  
+        - duration counts of new clients 
+  - Trend chart of storage deals
+    - abscissa: Time(24hour,Day,Week,Month,Year)
+    - ordinate:
+      - The line chart:  dealed price (min,max,avg,start,end)
+      - histogram:  
+        - duration active clients counts
+        - duration dealed capacity
+        - duration dealed counts
+        - duration dealed time cost
+  - Storage deal Size Distribution
+    - The pie chart
+  - Trend chart of storage ask
+    - abscissa: Time(24hour,Day,Week,Month,Year)
+    - ordinate:
+      - The line chart:  ask price (min,max,avg,start,end)
+      - histogram:  
+        - duration active miners counts
+        - duration dealed capacity
+- 5 RetrievalMarket
+  - Trend chart of retrieval miners ，power&collateral
```

Comment:
This is an assumption about the retrieval market -- we won't have retrieval miners with power on mainnet launch.

Reply:
I'll update the PR, --we won't have retrieval miners with power on mainnet launch.